src/main/java/mate/academy/entity/Book.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.entity;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private int id;
```

Comment:
The `id` field should be of type `Long` instead of `int` to match the task requirements.